src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -476,6 +480,26 @@ public void testGetUriByFilenameFindsResourceRelativeToRootClasspath() throws Ex
                 content, startsWith("good"));
     }
 
+    @Test
+    public void testGetUriByFilenameClasspathPrefixLoadConfig() throws Exception {
+        final String filename = CommonUtil.CLASSPATH_URL_PREFIX
+            + getPackageLocation() + "/InputCommonUtilTestWithChecks.xml";
+        final URI uri = CommonUtil.getUriByFilename(filename);
+
+        final Properties properties = System.getProperties();
+        final Configuration config = ConfigurationLoader.loadConfiguration(uri.toString(),
+            new PropertiesExpander(properties));
+        assertEquals("Checker", config.getName(), "Unexpected config name!");
+    }
+
+    @Test
+    public void testGetUriByFilenameFindsRelativeResourceOnClasspathPrefix() throws Exception {
+        final String filename = CommonUtil.CLASSPATH_URL_PREFIX
+            + getPackageLocation() + "/InputCommonUtilTest_empty_checks.xml";
+        final URI uri = CommonUtil.getUriByFilename(filename);
+        assertThat("URI is null for: " + filename, uri, is(not(nullValue())));
```

Comment:
lets load config by content.

Reply:
As I've done for `testGetUriByFilenameClasspathPrefixLoadConfig()` above?

Reply:
yes, as validation for null should be avoided.
Sorry that we have bunch of not tests around as example.

Reply:
Done, I've also refactored existing tests in separate commit.